data-generation/src/main/scala/thesis/SnapshotDelta.scala

```diff
@@ -329,6 +336,14 @@ object SnapshotDeltaObject {
     })
   }
 
+  /** Returns the closest graph for a given instant
+   * Currently unused but might be relevant later
+   *
+   * @param instant   the timestamp
+   * @param snapshot1 snapshot1
+   * @param snapshot2 snapshot 2
```

Comment:
```suggestion
   * @param snapshot1 snapshot1
   * @param snapshot2 snapshot2
```
before my ocd kills me 😂

Reply:
haha disse scaladockommentarene får dessverre veldig lite kjærlighet